.eslintrc

```diff
@@ -39,13 +39,51 @@
     ],
```

Comment:
Because extends loads it automatically, pretty much. Since it's redundant I don't think it's important to add. We add it for `emotion` because [emotion doesn't expose anything extensions/recommended set of rules/etc.](https://emotion.sh/docs/@emotion/eslint-plugin). I think that is clearer than adding other ones that we do not *need* to add.